docs/guides/request_storage.mdx

```diff
@@ -0,0 +1,136 @@
+
+:::warning
+
+The <ApiLink to="class/RequestList">`RequestList`</ApiLink> class is in its early version and is not fully
+implemented. It is currently intended mainly for testing purposes and small-scale projects. The current
+implementation is only in-memory storage and is very limited. It will be (re)implemented in the future.
+For more details, see the GitHub issue [crawlee-python#99](https://github.com/apify/crawlee-python/issues/99).
+For production usage we recommend to use the <ApiLink to="class/RequestQueue">`RequestQueue`</ApiLink>.
+
+:::
+
+The following code demonstrates the usage of the <ApiLink to="class/RequestList">`RequestList`</ApiLink>:
+
+<Tabs groupId="request_list">
+    <TabItem value="request_list_basic_example" label="Basic usage" default>
+        <CodeBlock className="language-python">
+            {RlBasicExample}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="request_list_with_crawler" label="Usage with Crawler">
+        <CodeBlock className="language-python">
+            {RlWithCrawlerExample}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+{/*
+
+## Which one to choose?
+
+TODO: write this section, once https://github.com/apify/crawlee-python/issues/99 is resolved
+
+*/}
+
+## Request-related helpers
+
+We offer several helper functions to simplify interactions with request storages:
+
+- The <ApiLink to="class/AddRequestsFunction">`add_requests`</ApiLink> function allows you to manually add specific URLs to the configured request storage. In this case, you must explicitly provide the URLs you want to be added to the request storage. If you need to specify further details of the request, such as a `label` or `user_data`, you have to pass instances of the <ApiLink to="class/Request">`Request`</ApiLink> class to the helper.
+- The <ApiLink to="class/EnqueueLinksFunction">`enqueue_links`</ApiLink> function is designed to discover new URLs in the current page and add them to the request storage. It can be used with default settings, requiring no arguments, or you can customize its behavior by specifying URL selectors, choosing different enqueue strategies, or applying include/exclude filters to control which URLs are added. See [Crawl website with relative links](../examples/crawl-website-with-relative-links) example for more details.
```

Comment:
```suggestion
- The <ApiLink to="class/EnqueueLinksFunction">`enqueue_links`</ApiLink> function is designed to discover new URLs in the current page and add them to the request storage. It can be used with default settings, requiring no arguments, or you can customize its behavior by specifying link element selectors, choosing different enqueue strategies, or applying include/exclude filters to control which URLs are added. See [Crawl website with relative links](../examples/crawl-website-with-relative-links) example for more details.
```